src/OrchardCore.Modules/OrchardCore.Redis/Options/RedisKeyManagementOptionsSetup.cs

```diff
@@ -77,4 +81,46 @@ private void ChangeXmlRepositoryRedisKey(string redisKey)
             }
         }
     }
+
+    private void ValidateRedisServer()
+    {
+        if (_logger.IsEnabled(LogLevel.Warning) && !CheckRedisPersistenceEnabled())
```

Comment:
You can remove the `ValidateRedisServer` method and just put the if condition on line 38.